### PR TITLE
(DOCSP-4828): Update with k8s Operator.

### PR DESCRIPTION
**Pull request by atsansone**
@i80and : Can you check and see if I broke anything? It looks right. I [staged the change on the main landing page](https://docs-mongodbcom-staging.corp.mongodb.com/landing/anthonysansone/DOCSP-4828/).

@jdestefano-mongo : I am unsure if this should also go on the Tools or Cloud pages. Both are nice even 2x2 grids, etc. Maybe we can stay with just the one landing for now?

### CODE REVIEW

File: src/html/home.html [hunk 1]
```diff
@@ -233,6 +236,22 @@ <h4 class="block__title">
...
+      <div class="row">
+        <div class="col-4 no-left-gutter">
+          <div class="block">
+            <h3 class="block__header">Kubernetes</h3>
+
+            <h4 class="block__title">
+              <a class="block__link" href="https://docs.mongodb.com/kubernetes-operator/current/">MongoDB Enterprise Kubernetes Operator</a>
+            </h4>
+            <p class="block__body">
+              MongoDB Enterprise Kubernetes Operator provides deployment of containerized MongoDB instances using Kubernetes.
```
Comment (jdestefano-mongo) [inline src/html/home.html]: Thought: s / provides / enables

File: src/html/home.html [hunk 2]
```diff
@@ -54,6 +54,9 @@
               <li class="toc__item">
                 <a class="toc__link" href="https://docs.mongodb.com/spark-connector/current/">MongoDB Spark Connector</a>
               </li>
+              <li class="toc__item">
```
Comment (jdestefano-mongo) [inline src/html/home.html]: I'd vote to keep this in alphabetical order, placing this after Compass.

Comment (atsansone) [reply src/html/home.html]: Agreed. Fixed.

File: src/html/home.html (hunk 1 above)
Comment (atsansone) [reply src/html/home.html]: Agreed. Fixed.

**Comment by atsansone** [comment]
@jdestefano-mongo : Back to you! Thanks.

**Comment by atsansone** [comment]
[Staged](https://docs-mongodbcom-staging.corp.mongodb.com/landing/anthonysansone/DOCSP-4828/).

File: src/html/home.html [hunk 3]
```diff
@@ -233,6 +236,22 @@ <h4 class="block__title">
         </div>
       </div>
 
+      <div class="row">
+        <div class="col-4 no-left-gutter">
+          <div class="block">
+            <h3 class="block__header">Kubernetes</h3>
+
+            <h4 class="block__title">
+              <a class="block__link" href="https://docs.mongodb.com/kubernetes-operator/current/">MongoDB Enterprise Kubernetes Operator</a>
```
Comment (i80and) [inline src/html/home.html]: `/stable/`, not `/current/`

Comment (jdestefano-mongo) [reply src/html/home.html]: Good catch, thanks Andrew!

**Pull request merged**